library/src/main/java/com/okta/oidc/CustomTabOptions.java

```diff
@@ -0,0 +1,116 @@
+/*
+ * Copyright (c) 2019, Okta, Inc. and/or its affiliates. All rights reserved.
+ * The Okta software accompanied by this notice is provided pursuant to the Apache License,
+ * Version 2.0 (the "License.")
+ *
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0.
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *
+ * See the License for the specific language governing permissions and limitations under the
+ * License.
+ */
+
+package com.okta.oidc;
+
+import android.os.Parcel;
+import android.os.Parcelable;
+
+import androidx.annotation.AnimRes;
+import androidx.annotation.ColorInt;
+import androidx.annotation.RestrictTo;
+
+/**
+ * Custom tab options for color and animation transitions.
+ */
+@RestrictTo(RestrictTo.Scope.LIBRARY)
+public class CustomTabOptions implements Parcelable {
+    public CustomTabOptions() {
+    }
+
+    @ColorInt
+    private int mCustomTabColor = -1;
+    @AnimRes
+    private int mStartEnterResId = -1;
+    @AnimRes
+    private int mStartExitResId = -1;
+    @AnimRes
+    private int mEndEnterResId = -1;
+    @AnimRes
+    private int mEndExitResId = -1;
```

Comment:
Per this StackOverflow comment seems like R.anim ids tend to be negative: https://stackoverflow.com/questions/48835664/forcing-a-positive-integer-to-be-generated-from-ids-xml

I see them as negative in OV as well.

Reply:
Fixed

Reply:
Dont we have default behavior ? default color and animation ?

Reply:
The default is the Android system animations. The SDK does not provide any.